RNSound/RNSound.m

```diff
@@ -76,6 +76,7 @@ -(void) audioPlayerDidFinishPlaying:(AVAudioPlayer*)player
   if (key == nil) return;
 
   @synchronized(key) {
+    [self setOnPlay:NO forPlayerKey:key];
```

Comment:
`audioPlayerDidFinishPlaying` is called when `AVAudioPlayer` pause or stop? If not, it should be added in the `stop` and `pause`.